src/units/abilities.cpp

```diff
@@ -1168,52 +1168,137 @@ void attack_type::modified_attacks(unsigned & min_attacks,
 	}
 }
 
-//Functions used for change damage_type list with damage
-static std::optional<std::string> select_damage_type(const unit_ability_list& abil_list, const std::string& type)
+//functions for select damage type who must be returned for each attribute used in damage_type()
```

Comment:
Either each select function should be documented or this comment removed. To me it's enough if `select_damage_type` gets its improved description back.

Reply:
comment removed